ui/src/table.test.tsx

```diff
@@ -642,9 +642,10 @@ describe('Table.tsx', () => {
       fireEvent.click(getAllByText('Col2')[1]!)
       fireEvent.click(container.querySelector('.ms-GroupHeader-expand')!)
 
-      expect(getAllByRole('row')).toHaveLength(tableProps.rows.length + headerRow + groupByRow)
+      expect(getAllByRole('row')).toHaveLength(tableProps.rows!.length + headerRow + groupByRow)
       fireEvent.change(getByTestId('search'), { target: { value: cell21 } })
-      expect(getAllByRole('row')).toHaveLength(headerRow + groupByRow)
+      const filteredItems = tableProps.rows!.filter(row => row.cells.find(cell => cell === cell21)).length
```

Comment:
No need, we know there is just a single row with that value, you can hardcode it.

Reply:
Do you mean like this `expect(getAllByRole('row')).toHaveLength(headerRow + groupByRow + 1)`or should I declare a constant in the top of the file similar to `headerRow` or `groupByRow`so that someone in a future can easily find what   does value 1 stand for?